library/validate-cpp/v0.6.12/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1.4
+FROM golang:1.19.1-bullseye AS build
+WORKDIR /app
+COPY --link . /app
+RUN CGO_ENABLED=0 go install -ldflags "-s -w" -trimpath github.com/envoyproxy/protoc-gen-validate@v0.6.12
```

Comment:
This is because the module is still named like so?

https://github.com/bufbuild/protoc-gen-validate/blob/main/go.mod#L1

Do you know if there are any plans to change this, more a curiosity.